### ISSUE TITLE
Problems with libsoup

### ISSUE THREAD

**Issue by kirushyk** [valum-framework/valum#70]
I'm using new version of libsoup, Seems like Valum treat libsoup in old way and use old SoupServer Listening API. Am I wrong?
https://developer.gnome.org/libsoup/stable/SoupServer.html#soup-server-listen
Default port to listen can not be changed.
There are build time errors also:
    valum/src/status.vala:77.32-77.37: error: The name 'Status' does not exist in the context of 'Valum.Route.fire'
    valum/src/router.vala:44.18-44.28: error: The name 'Status' does not exist in the context of 
'Soup'
    valum/src/vsgi/fastcgi.vala:135.61-135.71: error: The name 'Status' does not exist in the context of 'Soup'

**Comment by kirushyk** [comment]
Oh, forget about second part: it seems I used old .vapi file for modern libsoup.

**Comment by arteymix** [comment]
By the way, it uses the newest implementation based on the available version of libsoup-2.4.

The default port can be changed if you pass the CLI arguments to the `run` function like:

```vala
public static int main (string[] args) {
    var app = new Router ();
    return new Server (app).run (args);
}
```

Or if you use a set of predefined arguments:

```vala
new Server (app).run ({"app", "--port", "3003"});
```

**Comment by arteymix** [comment]
In the dependencies section, it tells what version of what package enable additional features: http://valum-framework.readthedocs.org/en/latest/installation.html#dependencies

**Comment by kirushyk** [comment]
Oh, okay.
I tried new VSGI.SoupServer(app, 8080).run();

**Comment by arteymix** [comment]
I also changed that, I think it's preferable to have uniform interfaces for server implementation, it makes them easily interchangable:

```vala
using VSGI.Soup; // you can also use VSGI.FastCGI

new Server (app).run ();
```

By the way, I have released `0.1.0-alpha`, I recommend you to use it ;)
